db/migrate/20210726183045_change_to_utf8_mb4.rb

```diff
@@ -74,68 +44,8 @@ def self.up
     execute "ALTER TABLE users CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
     execute "ALTER TABLE users_also_views CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
     execute "ALTER TABLE votes CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
-
```

Comment:
All of these execs are because old copies of the database were just 2 byte utf8 and so the indexes had to be removed before the columns were converted to a better character set, and then added back (but with a shortened key length due to mysql limitations)  Not sure why you deleted all of these. there had been a small problem with this migration in the past but that was fixed.